ec2/sc-ec2-linux-jumpcloud-notebook.yaml

```diff
@@ -235,6 +247,8 @@ Resources:
           SetupJumpcloud:
             - install_jc
             - config_jc
+          SetupApacheProxy:
```

Comment:
You need to add this to `--configsets` here https://github.com/Sage-Bionetworks/service-catalog-library/blob/5ae7de8de55d02b31a5ae938d0736404e09b0f4d/ec2/sc-ec2-linux-jumpcloud-notebook.yaml#L269

Reply:
Sounds good- Should this script be able to handle the auto-reloader cloudformation hook? It looks like all boot scripts are hooked into that event, though this one would fail.

Reply:
what you've done will not work on a CFN update that replaces an instance.  i think we need to find a solution for this however this PR is also blocking @tthyer 's work so i think we can do a follow on PR for fix.

Reply:
I've updated the utils script to handled rewrites at any time... https://github.com/Sage-Bionetworks/service-catalog-utils/pull/11

That requires re-tagging a utils release, I think